pyproject.toml

```diff
@@ -10,7 +10,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
```

Comment:
Updated the python version because we actually do depend on 3.9 per some new type annotations.

Reply:
Yes, good catch